tests/SixLabors.Fonts.Tests/Issues/Issues_36.cs

```diff
@@ -15,7 +15,7 @@ public class Issues_36
         [InlineData(3)]
         [InlineData(4)]
         [InlineData(5)]
-        public void TextWidthFroTabOnlyTextSouldBeSingleTabWidthMultipliedByTabCount(int tabCount)
+        public void TextWidthFroTabOnlyTextShouldBeSingleTabWidthMultipliedByTabCount(int tabCount)
```

Comment:
Thanks for all the awesome fixes. Would you be so kind to change `Fro` into `For`?

Reply:
I just force-pushed this fix.